frontend/src/i18n/hu.json

```diff
@@ -2,14 +2,72 @@
   "login": {
     "buttontext": "Bejelentkezés",
     "connectionProblems": "Kapcsolati probléma",
-    "email": "Email",
+    "email": "Email cím",
     "invalidData": "Rossz email vagy jelszó",
     "invalidPassword": "A jelszó minimum 8 karakter",
     "invalidemail": "Hibás email",
     "noEmail": "Az email megadása kötelező",
     "password": "Jelszó",
-    "title": "Bejelentkezés"
+    "title": "Belépés"
   },
-  "registration": "registration",
-  "text": "5"
+  "registration": {
+    "title": "Regisztráció",
+    "fullName": "Név",
+    "privatePerson": "Magánszemély",
+    "legalPerson": "Jogi személy",
+    "taxNumber": "Adószám",
+    "password": "Jelszó",
+    "passwordConfirmation": "Jelszó mégegyszer",
+    "email": "Email cím",
+    "buttontext": "Registration",
+    "alreadyRegistered": "Regisztrált már?",
+    "agreement": "Elfogadom az",
+    "linkAszf": " ÁSZF-et",
+    "and": " és az",
+    "linkadat": " adatkezelési alapelveket",
+    "address": "Lakcím/Székhely",
+    "invalidemail": "Az email cím nem megfelelő",
+    "login": "Belépés"
+  },
+  "companyAdmin": {
+    "newProduct": "New product",
+    "edit": "Edit",
+    "remove": "Remove",
+    "search": "Search"
+  },
+  "superAdmin": {
+    "createNewUser": "Create a new user",
+    "name": "Name",
+    "active": "Active",
+    "inactive": "Inactive",
+    "dateOfRegistration": "Date of registration",
+    "editBasicData": "Edit basic data",
+    "editProducts": "Edit products",
+    "delete": "Delete",
+    "confirmDelete": "Confirm delete?",
+    "search": "Search"
+  },
+  "datasheet": {
+    "descripotion": "Description",
+    "Events": "Events",
```

Comment:
itt is legyen kisbetűs. használjátok az i18next-scannert ;)

Reply:
kijavítottam a typot, a scannert meg következő alkalommal futtatjuk, hogy ez már ne vesszen el